### ISSUE TITLE
Rearrange fridges

### ISSUE THREAD

**Issue by johnmckerrell** [DoESLiverpool/somebody-should#1072]
We now have four fridges following the kind donation of a larger glass fronted fridge. I propose that we put the new fridge in the events room and move the existing glass fronted fridge into the main room. The one in the main room would then be used for the soft drinks (only) allowing the white fridge to be used for everything else. The smaller glass fronted fridge could be located at a lower height and doesn't even need to be in the kitchen (although should still have a payment method handy). Not sure exactly where it would go. The small white fridge currently in the events room should then most likely be binned/donated elsewhere.

Just been reminded by Adrian that there's actually five fridges in the space as there's also a smaller glass fronted fridge owned by Find Out Digital! I'll assume that's going in their room at some point.

**Comment by ajlennon** [comment]
Where are we going to find enough beer and wine to fill all these 'fridges?

**Comment by johnmckerrell** [comment]
Seán and I did a bit of moving of fridges just now. We've emptied the Stella Artois branded one out into the large glass fronted one and then moved the Stella Artois one to next to the coffee (turning the small filing cabinet 90 degrees created some useful space there). I've left these fridges off for now as I believe you're supposed to leave fridges 24 hours after moving them. When the large one is on in the events room we can empty the white one into it and then hopefully can get rid of, or at least "store" the white one.

We're thinking that the area around the coffee machine can be the new sharing area and the sweets and water butt can live here now too, we'll see how well this all works.
